Wikipedia/Code/PageEditorViewController.swift

```diff
@@ -290,7 +301,17 @@ extension PageEditorViewController: SectionEditorNavigationItemControllerDelegat
     }
     
     func sectionEditorNavigationItemController(_ sectionEditorNavigationItemController: SectionEditorNavigationItemController, didTapCloseButton closeButton: UIBarButtonItem) {
-        delegate?.pageEditorDidCancelEditing(self, navigateToURL: nil)
+        
+        if navigationItemController.progressButton.isEnabled {
```

Comment:
I added better enabled button handling in https://github.com/wikimedia/wikipedia-ios/pull/4719, so this conditional should keep the alert limited to only a textview with changes once that PR is in.